chrome/script.browserify.js

```diff
@@ -149,3 +150,30 @@ function getLoginData() {
     }
   );
 }
+
+// This function uses regular DOM
+// therefore there is no need for redraw calls
+function keyHandler(e) {
+  switch (e.key) {
+  case 'ArrowUp':
+    switchFocus('button.login:last-child', 'previousElementSibling');
+    break;
+
+  case 'ArrowDown':
+    switchFocus('button.login:first-child', 'nextElementSibling');
+    break;
+  }
+}
+
+function switchFocus(firstSelector, nextNodeAttr) {
+  var inputId = 'search-field';
+  var newActive = document.activeElement.id === inputId ?
+      document.querySelector(firstSelector) :
```

Comment:
this is using querySelector. Compatibility should be probably fine due to browser selection. https://www.w3schools.com/jsref/met_document_queryselector.asp